docs/AboutUs.md

```diff
@@ -2,58 +2,59 @@
 layout: page
 title: About Us
 ---
+# About InternBuddy
+We are team InternBuddy, and we are a team based in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
 
-We are a team based in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
-
-You can reach us at the email `seer[at]comp.nus.edu.sg`
+You can reach us at the email `e0774066[at]comp.nus.edu.sg`
 
 ## Project team
 
-### John Doe
+### Christopher Tan Rui Yang
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/potty10.png" width="200px">
 
-[[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](https://github.com/potty10)]
+[[portfolio](team/potty10.md)]
 
-* Role: Project Advisor
+* Role: Team Lead
+* Responsibilities: Testing
 
-### Jane Doe
+### Eugene Tang Kangjie
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/eugenetangkj.png" width="200px">
 
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](http://github.com/eugenetangkj)]
+[[portfolio](team/eugenetangkj.md)]
 
-* Role: Team Lead
-* Responsibilities: UI
+* Role: Developer
+* Responsibilities: UI, Documentation
 
-### Johnny Doe
+### Koh Kai Xun 
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/kohkaixun.png" width="200px">
 
-[[github](http://github.com/johndoe)] [[portfolio](team/johndoe.md)]
+[[github](http://github.com/kohkaixun)] 
+[[portfolio](team/kohkaixun.md)]
 
 * Role: Developer
-* Responsibilities: Data
+* Responsibilities: Code quality
 
-### Jean Doe
+### Lim Hai Leong Shawn 
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/seadragon2000341.png" width="200px">
 
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](http://github.com/seadragon2000341)]
+[[portfolio](team/seadragon2000341.md)]
 
 * Role: Developer
-* Responsibilities: Dev Ops + Threading
+* Responsibilities: Scheduling and Tracking, Deliverables and deadlines
```

Comment:
Inconsistency for naming, should `deadlines` be capitalised?

Reply:
Sounds good to me.